content/en/profiler/enabling/dotnet.md

```diff
@@ -81,10 +85,22 @@ Otherwise, install the profiler using the following steps, depending on your ope
 </div>
 
 
-You can install the Datadog .NET Profiler machine-wide so that all services on the machine can be instrumented, or you can install it on a per-application basis to allow developers to manage the instrumentation through the application's dependencies. To see machine-wide installation instructions, click the **Windows** or **Linux** tab. To see per-application installation instructions, click the **NuGet** tab.
+You can install the Datadog .NET Profiler machine-wide so that any services on the machine can be instrumented, or you can install it on a per-application basis to allow developers to manage the instrumentation through the application's dependencies. To see machine-wide installation instructions, click the **Windows** or **Linux** tab. To see per-application installation instructions, click the **NuGet** tab.
 
 {{< tabs >}}
 
+{{% tab "Linux with Single Step APM Instrumentation" %}}
+1. With [Single Step APM Instrumentation][1], there is nothing else to install. Go to [Enabling the Profiler](#enabling-the-profiler) to see how to activate the profiler for an application.
+
+<div class="alert alert-warning">
+  <strong>Note:</strong> If APM was already manually installed, it is required to uninstall it by removing the following environment variables: CORECLR_ENABLE_PROFILING, CORECLR_PROFILER, CORECLR_PROFILER_PATH, and the value that points to Datadog.Linux.ApiWrapper.x64.so in LD_PRELOAD. If they are in your dockerfile, remove them.
```

Comment:
This could be a dedicated `upgrading from a manual installation to single step instrumentation`? This can be shared between tracing and profiling ?
Not a blocking comment as your documentation is already an improvement.

Reply:
Let's see in another PR if the "upgrade" scenario could be documented at the APM level or in both tracer and profiler